log/logger.go

```diff
@@ -27,39 +29,15 @@ func init() {
 }
 
 // ModuleKey defines a module logging key.
-const ModuleKey = "module"
+const ModuleKey = corelog.ModuleKey
```

Comment:
Imo there is no need to have logger import core, feels like adding an edge to the dependency graph for no real reason.

Reply:
Logger needs to import core, due to `With(...) Logger`. Otherwise, it doesn't fulfill the interface.